tex/gregoriotex-signs.tex

```diff
@@ -1664,14 +1664,14 @@
   \ifnum#4=0\relax %
     % we try to avoid line breaking after a flat or a natural
     \GreNoBreak %
-    \gre@skip@temp@four = \gre@skip@alterationspace\relax%
+    \gre@skip@temp@four = \gre@dimen@alterationspace\relax%
     \ifgre@firstglyph%
       \gre@debugmsg{bolshift}{making adjustments for leading alteration}%
       \global\advance\gre@dimen@notesaligncenter by \wd\gre@box@temp@width %
       \global\advance\gre@dimen@notesaligncenter by \dimexpr\gre@skip@temp@four\relax %
       \kern\gre@skip@temp@four %
       \global\gre@dimen@bolextra = \wd\gre@box@temp@width%
-      \gre@skip@temp@four = \gre@skip@alterationspace\relax%
+      \gre@skip@temp@four = \gre@dimen@alterationspace\relax%
```

Comment:
Not your fault, but I don't think this line is needed at all.  It's a repeat of line 1667 and `\gre@skip@temp@four` hasn't been changed in the half-dozen lines in between.